src/sidebar/components/Excerpt.js

```diff
@@ -38,12 +38,13 @@ function InlineControls({ isCollapsed, setCollapsed, linkStyle = {} }) {
     >
       <div className="flex justify-end">
         <LinkButton
-          classes="inline underline"
+          color="text"
```

Comment:
`LinkButton`s now have "colors", not "variants." The former "dark" variant maps to `color="text"`. This is documented in the pattern library.